src/common/hooks/useBreakpoint.ts

```diff
@@ -0,0 +1,12 @@
+/* Determines if the screen width is less than a specified width */
+
+import { useMediaQuery } from '@material-ui/core';
+import { useTheme } from '@material-ui/core/styles';
+
+export default function useBreakpoint(breakpointWidth: number): boolean {
+  const theme = useTheme();
+  const isBelowBreakpoint = useMediaQuery(
+    theme.breakpoints.down(breakpointWidth),
+  );
+  return isBelowBreakpoint;
+}
```

Comment:
gonna sprinkle this around + replace the many instances where ive written/repeated the logic here